public/collections.json

```
@@ -1,4 +1,28 @@
 {
+  "Generative AI": {
+    "id": "genAI",
```

If we reflect the casing of our repositories names, this should be:
```suggestion
    "id": "GenAI",
```